components/integration/integration.php

```diff
@@ -418,6 +476,22 @@ public function order_status( $order_id, $status_old = 'new', $status_new = 'pen
 				if ( function_exists( $subscription_function ) ) {
 					$response = call_user_func( $subscription_function, $subscription_id, $email, $name );
 
+					if ( ! is_wp_error( $response ) && ! empty( $this->api_key ) && 'tag' === $subscription_type ) {
+                        $options = ckwc_get_subscription_options();
+                        $tags    = array();
+                        foreach ( $options as $option ) {
+	                        if ( 'tag' !== $option['key'] ) {
+		                        continue;
+	                        }
+                            $tags = $option['options'];
+                        }
+                        if ( $tags ) {
+	                        if ( isset( $tags[ $subscription_id ] ) ) {
+	                            wc_create_order_note( $order_id, sprintf( __( '[ConvertKit] Customer tagged with: %s', 'woocommerce-convertkit' ), $tags[ $subscription_id ] ) );
+	                        }
+                        }
+                    }
+
 					$debug = $this->get_option( 'debug' );
```

Comment:
I know it's outside the scope of this PR. But if $debug is only used in the if() condition, it shouldn't be a variable.

Reply:
👍 no reason not to clean up as we go

Reply:
To be clear you're suggesting we could just do this, right?
```
if ( 'yes' === $this->get_option( 'debug' ) )
```